syntax = "proto3";  // 指定使用proto3语法

package yh_bot;    // 定义包名，防止命名冲突

    
// 响应状态
message Status {
    int64 request_id = 1;
    int32 code = 2;
    string msg = 3;
}

// 获取机器人信息
message bot_info_send {
    string id = 2;
}

message bot_info {
    Status status = 1;
    Bot_data data = 2;
    message Bot_data {
        string bot_id = 1;
        string name = 2;
        int64 name_id = 3; // 名称ID
        string avatar_url = 4;
        string avatar_id = 5;
        string introduction = 6;
        string create_by = 7;
        int64 create_time = 8;
        int64 headcount = 9; // 使用人数
        int32 private = 10;
        int32 is_stop = 11; // 是否停用
        int32 always_agree = 13; // 自动进群
        int32 do_not_disturb = 15; // 免打扰
        int32 top = 18; // 置顶
        int32 group_limit = 20; // 限制进群
    }
}